apps/web/src/pages/ApplicantDashboardPage/components/ReviewTalentRequestDialog.tsx

```diff
@@ -0,0 +1,409 @@
+/* eslint-disable formatjs/no-literal-string-in-jsx */
```

Comment:
Maybe we want to place this exception on the four different locations instead of for the whole file?  This way we won't accidentally mask other errors.

Reply:
Good call :+1: Changed in [Move eslint-disable above each line](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14135/commits/384d6b13795cb5c1d999fd81f73dc45bd8125624)